modules/core/src/test/scala/fs2/kafka/KafkaConsumerSpec.scala

```diff
@@ -66,7 +66,7 @@ final class KafkaConsumerSpec extends BaseKafkaSpec {
       }
     }
 
-    it("should consume all records with subscribing for several consumers") {
+    it("should consume all records at least once with subscribing for several consumers") {
```

Comment:
This was flaking before (#486) but changing to foralltestcontainer made it fail consistently. I think we should change the test to not expect exactly-once delivery.